roles/splunk_common/tasks/remove_clustering.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Remove indexer clustering stanza in order to reconfigure
+  ini_file:
+    dest: "{{ splunk.home }}/etc/system/local/server.conf"
```

Comment:
So this is assuming that the only place where we have clustering stanza is in etc/system/local/server.conf and not at any other place like etc/slave-apps/ ?

Reply:
Yes we are dealing with the etc/server/local.conf only with this change.